providers/aws/resources/aws_s3.go

```diff
@@ -164,13 +164,25 @@ func (a *mqlAwsS3Bucket) id() (string, error) {
 	return a.Arn.Data, nil
 }
 
-func (a *mqlAwsS3Bucket) policy() (*mqlAwsS3BucketPolicy, error) {
-	bucketname := a.Name.Data
-
-	location := a.Location.Data
+func emptyAwsS3BucketPolicy(runtime *plugin.Runtime) (*mqlAwsS3BucketPolicy, error) {
+	res, err := CreateResource(runtime, "aws.s3.bucket.policy", map[string]*llx.RawData{
+		"name":       llx.StringData(""),
+		"document":   llx.StringData("{}"),
+		"version":    llx.StringData(""),
+		"id":         llx.StringData(""),
+		"statements": llx.ArrayData([]interface{}{}, types.Dict),
```

Comment:
adding these as empty pre-cached fields so the runtime doesn't attempt to compute them